Model/ArticleInterface.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ArticleBundle\Model;
+
+use Doctrine\Common\Collections\Collection;
+use Sonata\MediaBundle\Model\Media;
+
+/**
+ * Interface ArticleInterface.
+ *
+ * @author Benoit Mazière <bmaziere@ekino.com>
+ */
+interface ArticleInterface
+{
+    /**
+     * Get id.
+     *
+     * @return mixed
+     */
+    public function getId();
+
+    /**
+     * Set id.
+     *
+     * @param mixed $id
+     *
+     * @return $this
+     */
+    public function setId($id);
+
+    /**
+     * Set abstract.
+     *
+     * @param $abstract
+     *
+     * @return $this
+     */
+    public function setAbstract($abstract);
+
+    /**
+     * Get abstract.
+     *
+     * @return string $abstract
+     */
+    public function getAbstract();
+
+    /**
+     * Set categories.
+     *
+     * @param Collection $categories
```

Comment:
A `Collection` of what? It should be precised.

Examples:

```
int[]|Collection
```

Or:

```
Category[]|Collection
```

Same for the above.